src/app/search/directives/search-form.component.css

```diff
@@ -0,0 +1,33 @@
+form {
+  display: block;
+}
+
+input {
+  width: 100%;
+  border: 0;
+  padding: 6px;
+  color: #939393;
+  background: #fff;
+}
+
+option[disabled] {
+  display: none;
+}
+
```

Comment:
When moving this .form-group stuff into here, I noticed these two css files were identical and consolidated.